src/admin/class-admin.php

```diff
@@ -21,6 +20,26 @@
  */
 class Admin implements Service {
 
+  /**
+   * Manifest Data Interface.
```

Comment:
You missed this comment from my earlier review 🙂 

This is not an interface but an instance of the `Manifest` object (or what ever object you are passing to it).